src/GeositeFramework/js/Plugin.js

```diff
@@ -235,8 +235,10 @@ require(['use!Geosite',
                     showValueKey = pluginObject.toolbarName + " showinfographic";
                 if (typeof localStorage[showValueKey] !== 'undefined') {
                     return localStorage[showValueKey] === 'true';
-                }
-                return true;
+                } else {
+					localStorage.setItem(showValueKey,pluginObject.showInfographicOnStart);
+					return localStorage[showValueKey] === 'true';
+				}
```

Comment:
I tested out @kdeloach's suggestion a few times, and here is the behavior I'm seeing.

If the user unchecks the "Don't show this on start" option after clicking the "?" button on a plugin, the local storage value is set and the infographic opens the next time the plugin is activated. If the user then rechecks the option in the infographic, the local storage value is updated, and the infographic isn't shown again until the "?" is clicked.

If the user never interacts with the "Don't show this on start" option, the local storage option is never set, and the plugin override value is used.

The change @kdeloach suggested also has the side benefit of not overwriting the user's preferred settings. If they have previously hid the infographic, that setting will not be overridden by the plugin setting.